inc/saifdb.h

```diff
@@ -105,6 +107,7 @@ typedef enum _sai_fdb_entry_attr_t
     SAI_FDB_ENTRY_ATTR_META_DATA,
 
     /* -- */
+    SAI_FDB_ENTRY_ATTR_END,
 
     /* Custom range base value */
     SAI_FDB_ENTRY_ATTR_CUSTOM_RANGE_BASE  = 0x10000000
```

Comment:
Zubin, good suggestion. I considered that but did not add for now as there is no custom stuff in yet and I did not find a very clear guideline. My target was more towards getting the maximum attribute count needed for saiobject.h. Let me check if I can update this quickly. 
@lguohan please merge this if this is ok as it will expedite our implementation greatly.